shell/src/main/java/org/apache/accumulo/shell/commands/PingCommand.java

```diff
@@ -31,31 +31,37 @@
 
 public class PingCommand extends Command {
 
-  private Option tserverOption, disablePaginationOpt;
+  private Option serverOption, tserverOption, disablePaginationOpt;
```

Comment:
Tested this with uno. The `-ts` flag option has the same usage as `-s` 
```
root@uno> ping --help
usage: ping [-?] [-np] [-s <server address>] [-ts <tablet server>]
description: ping compactors, scan servers, or tablet servers
  -?,--help                              display this help
  -np,--no-pagination                    disable pagination of output
  -s,--server <server address>           compactor, scan server, or tablet server address to ping
  -ts,--tabletServer <tablet server>     compactor, scan server, or tablet server address to ping
```
I'm able to pass a compactor `host:port` string with either the `-ts` or `-s` option flags and they both work.
```
[lock.ServiceLock] DEBUG: [zlock#1aa7a951-9a79-498c-8f74-69cf3c6f4469#] Ephemeral node /accumulo/e4262575-b573-48cc-b4ae-54c2284dff33/compactors/default/localhost:9133/zlock#1aa7a951-9a79-498c-8f74-69cf3c6f4469#0000000000 created with data: {"descriptors":[{"uuid":"1aa7a951-9a79-498c-8f74-69cf3c6f4469","service":"CLIENT","address":"localhost:9133","group":"default"},{"uuid":"1aa7a951-9a79-498c-8f74-69cf3c6f4469","service":"COMPACTOR","address":"localhost:9133","group":"default"}]}

root@uno> ping -ts localhost:9133
localhost:9133 OK
root@uno> ping -s localhost:9133
localhost:9133 OK
```
Given that, does the `-ts` flag still need to exist or can it just be replaced with `-s`?